tls/s2n_prf.c

```diff
@@ -325,7 +331,8 @@ int s2n_prf_free(struct s2n_connection *conn)
     return conn->prf_space.tls.p_hash_hmac_impl->free(&conn->prf_space);
 }
 
-static int s2n_prf(struct s2n_connection *conn, struct s2n_blob *secret, struct s2n_blob *label, struct s2n_blob *seed_a, struct s2n_blob *seed_b, struct s2n_blob *out)
+static int s2n_prf(struct s2n_connection *conn, struct s2n_blob *secret, struct s2n_blob *label, struct s2n_blob *seed_a,
+                   struct s2n_blob *seed_b, struct s2n_blob *seed_c, struct s2n_blob *out)
 {
     if (conn->actual_protocol_version == S2N_SSLv3) {
```

Comment:
It looks wrong that seed_c is silently dropped for sslv3_prf even though we never plan to use 3 parts with that version of the protocol.

Reply:
I propagated seed_c everywhere seed_b goes.